src/types/Plan.ts

```diff
@@ -47,8 +48,11 @@ export interface AddOn {
 export interface PricingPlan {
     billing_cycle: BillingCycle;
     currency: Currency;
-    amount: number;
+    billing_schema: BillingSchema;
```

Comment:
Just noticed this when testing, but API returns this as `billing_scheme` and not `billing_schema`. Not sure which form is the correct one so I'll let you fix it.

Reply:
I see: https://github.com/prezly/javascript-sdk/pull/339